util/buildinfo.go

```diff
@@ -6,4 +6,6 @@ var (
 	Version = "undefined"
 	// BuildTime build time of the binary
 	BuildTime = "undefined"
+	// Architecture current CPU architecture
+	Architecture = "amd64"
```

Comment:
Having `amd64` as the default is surprising IMO. Can we provide the value via the `Makefile` instead like for the other vars?  
`go env GOARCH` should give the correct output.

Reply:
Yes that's correct.
I will change it. 👍